### DISCUSSION TITLE
Can't modify "SystemControlPageBackgroundChromeLowBrush" used in SplitView control

### DISCUSSION THREAD

**Discussion by creamIcec** [AvaloniaUI/Avalonia#14611] [Q&A] [answered]
When I was trying to set the color of pane inside SplitView to transparent, it ran into some issues. It seems whatever I use Style or overriding the DynamicResource, nothing works.
Below is my process:
Firstly, I opened devtools to get the name of pane:
![图片](https://github.com/AvaloniaUI/Avalonia/assets/42510470/5824ed9b-dfe7-4cbe-83bf-23d21de1ce73)
And set its style as below:
```xaml
<Style Selector="SplitView /template/ Panel#PART_PaneRoot">
  <Setter Property="Background" Value="Transparent"/>
</Style>
```
But this didn't work, so I turned to check if it uses dynamic resources, and it does:
https://github.com/AvaloniaUI/Avalonia/blob/0c2c3f1aeb01bb8a7661d206a27065a6ac38aac3/src/Avalonia.Themes.Fluent/Controls/SplitView.xaml#L28
So I turn to override the DynamicResources value:
```xaml
<SolidColorBrush x:Key="SystemControlPageBackgroundChromeLowBrush">Transparent</SolidColorBrush>
```
But this also not worked, Wherever I put this inside, UserControl.Resources or SplitView.Resources.
What's the problem? Please help me and thank you in advance!

Environment Infomation:
* Avalonia 11.0.4
* .NET 7.0

**Comment by ykarpeev** [comment]
This works for me with Avalonia 11.0.9 if I set to Transparent. Here is an example with yellow.


```
 <SplitView
       DisplayMode="Inline"
       IsPaneOpen="True"
       OpenPaneLength="300">
        <SplitView.Resources>
               <ResourceDictionary>
                   <SolidColorBrush x:Key="SystemControlPageBackgroundChromeLowBrush">Yellow</SolidColorBrush>
               </ResourceDictionary>
           </SplitView.Resources>
       <SplitView.Pane>
           <TextBlock
               HorizontalAlignment="Center"
               VerticalAlignment="Center"
               FontSize="24"
               Text="Pane" />
       </SplitView.Pane>

       <Grid>
           <TextBlock
               HorizontalAlignment="Center"
               VerticalAlignment="Center"
               FontSize="24"
               Text="Content" />
       </Grid>
   </SplitView>
```

![image](https://github.com/AvaloniaUI/Avalonia/assets/6986975/7cbb5822-feca-4ec5-bdf1-cb65f2eb18b3)

**Reply by creamIcec** [reply]
After some experiments, I found that it's due to my application theme settings, as I'm using FluentAvalonia Theme globally:
```xaml
<Application.Styles>
  <sty:FluentAvaloniaTheme PreferUserAccentColor="True"/>
</Application.Styles>
```
and it prevents the SplitView from changing its styles. After finding this, I tried to apply a different theme(preset FluentTheme, which allows me changing the background) to the SplitView or its UserControl parent, and the former didn't work, the later did worked, but it will change whole theme of the view, which is not expected.
for SplitView I added:
```xaml
<SplitView.Styles>
  <FluentTheme/>
</SplitView.Styles>
```
and for UserControl:
```xaml
<UserControl.Styles>
  <FluentTheme/>
</UserControl.Styles>
```
So this may be an issue of FluentAvalonia Theme, not Avalonia.
Thank you for your effort!

**Reply by creamIcec** [reply]
From my view, seems currently the only way to achieve my goal is copy and paste everything of SplitView source code and only modify the property setter inside `<ControlTheme>`. This is redundant and not ideal.

**Reply by timunie** [reply, marked as answer]
if you use FluentAvalonia, you need to look up their resource keys to override. They have different names.

**Reply by creamIcec** [reply]
This works, Thank you for your help!
For anyone who interest in, the style for background:
https://github.com/amwx/FluentAvalonia/blob/9e175132935ede0103b6e21f7c4631d1c3133009/src/FluentAvalonia/Styling/ControlThemes/BasicControls/SplitViewStyles.axaml#L28
